omnigibson/object_states/particle_modifier.py

```diff
@@ -256,6 +255,24 @@ def is_compatible_asset(cls, prim, **kwargs):
 
         return True, None
 
+    @classmethod
+    def postprocess_ability_params(cls, params):
+        """
+        Post-processes ability parameters to ensure the system names (rather than synsets) are used for conditions.
+        """
+        # Import here to avoid circular imports
+        from omnigibson.utils.bddl_utils import get_system_name_by_synset
+
+        for sys in list(params["conditions"].keys()):
```

Comment:
shouldn't this be wrapped with `if "conditions" in params`?

Reply:
That was what I did initially, but then I realized conditions is a required positional arg for the particlemodifier class. So that key has to exist?

Reply:
gotcha, sg!